src/controllers/playback/video/index.js

```diff
@@ -579,6 +578,7 @@ export default function (view) {
         }, state);
         Events.on(player, 'playbackstart', onPlaybackStart);
         Events.on(player, 'playbackstop', onPlaybackStopped);
+        Events.on(player, 'promptskip', onPromptSkip);
```

Comment:
Same here regarding using the constant values. (The other events should probably be updated in a future cleanup PR)